config/locales/it.yml

```diff
@@ -343,129 +377,277 @@ it:
-      reported_user_body: 'Lei ha riferito %{reported_name}. Questo sarà esaminato da un amministratore.'
+      reported_user_body: 'Lei ha riferito %{reported_name}. Questo sarà
+      esaminato da un amministratore.'
       reportee_user_subject: 'if me | Sei stato segnalato'
-      reportee_user_body: 'Sei stato segnalato. Questo sarà esaminato da un amministratore.'
+      reportee_user_body: 'Sei stato segnalato. Questo sarà esaminato da un
+      amministratore.'
       add_ban_subject: 'if me | Sei stato escluso'
-      add_ban_body: 'Sei stato bannato per aver violato la nostra %{code_of_conduct_link}.'
+      add_ban_body: 'Sei stato bannato per aver violato la nostra
+      %{code_of_conduct_link}.'
       remove_ban_subject: 'if me | Non sei più bannato'
-      remove_ban_body: 'In precedenza sei stato bannato per aver violato la nostra %{code_of_conduct_link}. Non sei più bannato.'
+      remove_ban_body: 'In precedenza sei stato bannato per aver violato la
+      nostra %{code_of_conduct_link}. Non sei più bannato.'
   omniauth:
     google: Google
     access_denied: 'Accesso Negato'
   pages:
     about:
       send_us: noi
       email_us: 'Manda pure una EMail a %{email_link}!'
-      main_message_one: '%{app_name} è una community dedicata a esperienze sulla salute mentale che incoraggia i propri utenti a condividere le loro storie con alleati di cui ci si può fidare.'
-      message_text_one: 'Gli alleati sono le persone con cui interagiamo quotidianamente includendo amici, familiari, colleghi, professori ed esperti di salute mentale.'
-      main_message_two: 'Avere a che fare con la salute mentale è ciò che ci rende umani. Ma per molti di noi, essere aperti al riguardo è spesso molto difficile.'
-      message_text_two: '<p>Non tutti sono terapeuti. Le persone con cui abbiamo a che fare ogni giorno danno forma alle nostre emozioni e comportamenti. Renderli partecipi nel trattamento della salute mentale è la chiave per la cura.</p><p class="subtitle">"Non è solo nella tua testa, ma tutto attorno a te. Possiamo raggiungere una soluzione, insieme."</p><p>La salute mentale è un''esperienza individuale ed unica. Al mondo non esistono due menti uguali! Affrontare il problema richiede l''aiuto di tutti, motivo per cui il progetto è open source.</p>'
+      main_message_one: '%{app_name} è una community dedicata a esperienze
+      sulla salute mentale che incoraggia i propri utenti a condividere le loro
+      storie con alleati di cui ci si può fidare.'
+      message_text_one: 'Gli alleati sono le persone con cui interagiamo
+      quotidianamente includendo amici, familiari, colleghi, professori ed
+      esperti di salute mentale.'
+      main_message_two: 'Avere a che fare con la salute mentale è ciò che ci
+      rende umani. Ma per molti di noi, essere aperti al riguardo è spesso molto
+      difficile.'
+      message_text_two: '<p>Non tutti sono terapeuti. Le persone con cui abbiamo
```

Comment:
Does this work? I would toggle the languages in your local dev environment to actually confirm. Please do this for the other languages!

I think you'll need to use `>-`.